src/geofabrics/lidar.py

```diff
@@ -15,24 +15,33 @@
 
 
 class CatchmentLidar:
-    """ A class to manage LiDAR data in a catchment context by supporting the addition of LiDAR data tile by tile.
+    """ A class to manage LiDAR data in a catchment context by supporting the addition of LiDAR data tile by tile. LiDAR
+    files are loaded using PDAL and the point cloud, and it's extents are returned.
     """
     LAS_GROUND = 2
```

Comment:
Can this value change depending of the type of data?

Reply:
No this is specified in the LAS/LAZ format. I'll add a comment to that effect :)